### ISSUE TITLE
Support for automic hbase counters

### ISSUE THREAD

**Issue by RajatGoyal** [barseghyanartur/starbase#21]
Are there any plans of adding support for [counters](https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/util/Counter.html) in starbase.

**Comment by barseghyanartur** [comment]
@RajatGoyal:

No. At least, not in the nearest future, since I'm very limited in time. Pull requests are very welcome, though.
